### PR TITLE
[15.0][MIG] sale_order_secondary_unit: Add migration script to adapt secod uom to variants

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT45597

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

File: sale_order_secondary_unit/migrations/15.0.2.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Set default sale secondary uom from template to variants
+    templates = env["product.template"].search([("sale_secondary_uom_id", "!=", False)])
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/migrations/15.0.2.0.0/post-migration.py]: Can you please do this in SQL?
```
UPDATE product_product pp
SET sale_secondary_uom_id = pt.sale_secondary_uom_id
FROM product_template pt
WHERE pt.id = pp.product_tmpl_id
AND pt.sale_secondary_uom_id IS NOT NULL
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Done!!

**Pull request merged**